src/core/api_client/tests/factories/auth.factory.ts

```
@@ -31,4 +41,51 @@ export class AuthFactory {
+    });
+  }
+
+  static mockLogout(logoutEndpoint: string) {
+    fetchMock.post(logoutEndpoint, {
+      status: 200,
+      body: {},
+    });
+  }
+
+  static setupAuthClient(client: any, config: TokenConfig = {}) {
+    const token = this.createToken(config);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)